### PR TITLE
Additional Pentest Doc Updates

### PR DESCRIPTION
**Pull request by bwillis**

### CODE REVIEW

File: docs/organizations/pentest-phases.md [hunk 1]
```diff
@@ -32,13 +49,13 @@
...
-        <td>Create and save scoping drafts, collaborating seamlessly with team members.</td>
-        <td>Once approved, initiate the pentest setup on the HackerOne platform, addressing key questions.</td>
+        <td><ul><li>Create and save scoping drafts</li><li>collaborating seamlessly with team members.</li></ul></ul></td>
+        <td><ul><li>Upon approval, quickly start pentest setup on HackerOne, addressing key questions</li><li>Tailor your pentest workflows using platform integrations, triggers, and APIs.</li></ul></td>
     </tr>
     <tr style="background-color: #D1E5C9">
         <td>HackerOne Tasks</td>
-        <td>The HackerOne team evaluates your assets to determine the appropriate pentest size and provide a tailored quote. </td>
-        <td>With prepped assets and set pentester rewards, most tests can begin within days.</td>
+        <td><ul><li>We evaluate your assets to accurately determine the needed pentest size.</li><li>Receive a customized quote tailored to your specific pentest requirements.</li></ul></td>
```
[the remark this answers is not in the document]
Comment (bwillis) [reply docs/organizations/pentest-phases.md]: Thanks, updaged

File: docs/organizations/pentest-phases.md [hunk 2]
```diff
@@ -106,15 +123,15 @@
 </tbody>
 </table>
 
-![Pentest Engagement Process](./images/pentest-engagement-process.png)
+![Pentest Engagement Process](/images/pentest-engagement-process.png)
```
[the remark this answers is not in the document]
Comment (bwillis) [reply docs/organizations/pentest-phases.md]: Yeah, weird that this wasn't working in the local environment, but seems to work deployed so I'll leave it as is.

File: docs/organizations/retesting-pentest.md [hunk 3]
```diff
@@ -6,13 +6,10 @@ id: "organizations/retesting-pentest"
...
-|**Duration for Retesting**| Once claimed, a retest should be completed within 72 hours.                                                                                                                                                                                                                                                                                                   |
-|**Reporting Retests**| Any retesting findings or confirmations are documented in the initial vulnerability report. It's imperative for the pentesters to provide evidence, showcasing whether the vulnerability has been successfully addressed or still persists.                                                                                                                   |
-
-
-
+<table>
+<tr><td style="font-weight: bold" nowrap>Requesting Retesting</td><td>You may request retesting of specific bugs you've addressed during or after the pentest has concluded.<br /><br />Retests can be requested in the HackerOne platform. Please follow the guide [here](/organizations/retesting-pentest.html).                                                                                                           </td></tr>
```
[the remark this answers is not in the document]
Comment (bwillis) [reply docs/organizations/retesting-pentest.md]: Yeah, you are right, I did do a few already, but did miss some. Thanks!

File: docs/organizations/retesting-pentest.md [hunk 4]
```diff
@@ -6,13 +6,10 @@
...
+<tr><td style="font-weight: bold" nowrap>Retesting Window</td><td>The duration for which retesting can be requested without incurring extra costs varies based on the [pentest tier](/organizations/pentest-phases.html#tiers). It could be either 30 or 90 days post-pentest. To be clear on the time window for your specific tier, always refer to the pentest tier details.<br /><br />Each retest request is charged at $50 after the free retesting window concludes</td></tr>
```
[the remark this answers is not in the document]
Comment (bwillis) [reply docs/organizations/retesting-pentest.md]: Updated!

**Review by jessiwright — APPROVED** [review]

**Pull request merged**